website/docs/reference/widgets/code-scanner.md

```diff
@@ -5,6 +5,8 @@
 
 The Code Scanner widget allows you to scan a variety of barcodes and QR codes. Using this widget, you'll be able to enhance internal business operations and provide excellent customer service.
 
+<VideoEmbed host="youtube" videoId="Suhefwa5pz0" title="How To Build A Self-Checkout Payment System With The CodeScanner Widget" caption=""/>
```

Comment:
@megaconfidence The video appears to be private and is not available
![Uploading Screenshot 2023-02-09 at 15.53.12.png…]()